From 9ddc8a72981e9044e664d30b541c8329f501b850 Mon Sep 17 00:00:00 2001
From: Bahtiar `kalkin-` Gadimov <bahtiar@gadimov.de>
Date: Mon, 21 Aug 2017 12:23:02 +0200
Subject: [PATCH] Set default editor to vi

Upstream uses nano as editor, probably because it is default in Debian.
The default text editor in Fedora is vi.
---
 awesomerc.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/awesomerc.lua b/awesomerc.lua
index 5713b689..3d087531 100644
--- a/awesomerc.lua
+++ b/awesomerc.lua
@@ -48,7 +48,7 @@ beautiful.init(awful.util.get_themes_dir() .. "default/theme.lua")
 -- @DOC_DEFAULT_APPLICATIONS@
 -- This is used later as the default terminal and editor to run.
 terminal = "xterm"
-editor = os.getenv("EDITOR") or "nano"
+editor = os.getenv("EDITOR") or "vi"
 editor_cmd = terminal .. " -e " .. editor
 
 -- Default modkey.
-- 
2.13.2

